### ISSUE TITLE
Don't depend on GUI-Builders for the interface.

### ISSUE THREAD

**Issue by weisJ** [MyLibreLab/MyLibreLab#25]
Currently the GUI is build using the netbeans gui-builder which limits the choice of IDE for people working with the code.
Although the application will run fine in any other ide.

My proposition would be to slowly move the gui code away from gui-builders i.e. every time some view needs changes the
corresponding source file should be converted to be more humanly readable by rewriting the auto generated code.

Some preparatory steps to ensure consistency across the gui would be:
- Provide utility methods for
   - common components
   - borders
   - insets and spacing
   - resource files i.e. text, icons etc.

I'll add more tasks when I start working on the gui and have gotten a better overview of what needs to be done.

**Comment by marko-radosavljevic** [comment]
I completely agree. It's very unfortunate that the GUI is built using netbeans gui-builder.

Feel free to make any necessary GUI changes, you have my full support. Also add any issues you see, you can partition them in a way that is suitable for hacktober contributors ^^
